features.json

```diff
@@ -12,6 +12,7 @@
     "ember-container-inject-owner": true,
     "ember-htmlbars-local-lookup": null,
     "ember-application-engines": null,
-    "ember-glimmer": null
+    "ember-glimmer": null,
+    "ember-uniqBy": true
```

Comment:
Should generally be "package-name-feature-name", so in this case you would use ember-runtime-computed-uniq-by..